launcher/container_runner.go

```diff
@@ -240,12 +241,24 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 	if err != nil {
 		return nil, err
 	}
+	tokenWriter, err := models.NewFileWriter(launcherfile.HostTmpPath, launcherfile.AttestationVerifierTokenFilename)
+	if err != nil {
+		return nil, err
+	}
+
+	// Go < 1.23 only allows Reset to be called on stopped or expired timers with cleared channels.
+	// Create a timer that expires instantly and clear the chan as a workaround.
+	timer := models.NewRealTimer(0)
+	<-timer.C()
+
 	return &ContainerRunner{
 		container,
 		launchSpec,
 		attestAgent,
 		logger,
 		serialConsole,
+		tokenWriter,
+		timer,
```

Comment:
I feel like adding a timer in ContainerRunner just for the ease of test might be a little bit too much.

If we have to use a timer, maybe offloading the token writing step to the attestAgent might be better.